lib/mapObjects/CRewardableObject.h

```diff
@@ -343,6 +348,7 @@ class DLL_LINKAGE CRewardableObject : public CArmedInstance
 		h & static_cast<CArmedInstance&>(*this);
 		h & info;
 		h & canRefuse;
+		h & showInInfobox;
```

Comment:
FYI: generally, for minor changes like this you should:
1) Bump serialization version in CSerializer.h (but keep minimal version intact)
2) Serialize affected values in form
```
if ( h >= <BUMPED_VERSION>)
    h & showInInfobox;
else
    h = false;
```
In this case probably not important - will merge this PR alongside with your secondary skill changes, which bump minimal save format anyway.

Reply:
If I do not need to it now, so, let's resolve this conversation? 

But what structures is saved, and what not? I need to know it)

Reply:
> But what structures is saved, and what not? 

Generally - everything from library side gets saved. There are some exceptions, but those are mostly due to inability to serialized them (e.g. std::function's).

Yes, this is extensive, but this approach works fine. In long-term, dropping serialization of handlers and serializing only map itself might be a good idea, but this is definitely not something needed soon.

Reply:
done, serialization version bumped